.github/workflows/integration-tests.yml

```diff
@@ -327,75 +327,23 @@ jobs:
             os: ubuntu-latest
             run: -run TestOCRJobReplacement
             file: ocr
-            pyroscope_env: ci-smoke-ocr-evm-simulated
-          - name: ocr-geth
-            nodes: 1
-            os: ubuntu-latest
-            run: -run TestOCRBasic
-            file: ocr
-            client: geth
-            pyroscope_env: ci-smoke-ocr-evm-simulated         
-          # Uncomment, when https://smartcontract-it.atlassian.net/browse/TT-753 is DONE
-          # - name: ocr-nethermind
-          #   nodes: 1
-          #   os: ubuntu20.04-8cores-32GB
-          #   run: -run TestOCRBasic
-          #   file: ocr
-          #   client: nethermind
-          #   pyroscope_env: ci-smoke-ocr-evm-simulated                          
-          - name: ocr-besu
-            nodes: 1
-            os: ubuntu20.04-8cores-32GB
-            run: -run TestOCRBasic
-            file: ocr
-            client: besu
-            pyroscope_env: ci-smoke-ocr-evm-simulated        
-          - name: ocr-erigon
-            nodes: 1
-            os: ubuntu20.04-8cores-32GB
-            run: -run TestOCRBasic
-            file: ocr
-            client: erigon
-            pyroscope_env: ci-smoke-ocr-evm-simulated         
+            pyroscope_env: ci-smoke-ocr-evm-simulated                         
           - name: ocr2
             nodes: 1
             os: ubuntu20.04-8cores-32GB
             run: -run TestOCRv2JobReplacement
             file: ocr2
             pyroscope_env: ci-smoke-ocr2-evm-simulated
-          - name: ocr2-geth
-            nodes: 1
-            os: ubuntu20.04-8cores-32GB
-            run: -run TestOCRv2Basic
-            file: ocr2
-            client: geth
-            pyroscope_env: ci-smoke-ocr2-evm-simulated            
-          # Uncomment, when https://smartcontract-it.atlassian.net/browse/TT-753 is DONE            
-          # - name: ocr2-nethermind
-          #   nodes: 1
-          #   os: ubuntu20.04-8cores-32GB
-          #   run: -run TestOCRv2Basic
-          #   file: ocr2
-          #   client: nethermind
-          #   pyroscope_env: ci-smoke-ocr2-evm-simulated                         
-          - name: ocr2-besu
-            nodes: 1
-            os: ubuntu20.04-8cores-32GB
-            run: -run TestOCRv2Basic
-            file: ocr2
-            client: besu            
-            pyroscope_env: ci-smoke-ocr2-evm-simulated      
-          - name: ocr2-erigon
+          - name: ocr2
             nodes: 1
             os: ubuntu20.04-8cores-32GB
             run: -run TestOCRv2Basic
```

Comment:
shouldn't you remove that custom run directive? we want to run all tests in the file, not just this one